syncano_sync/scripts.py

```diff
@@ -0,0 +1,187 @@
+# coding=UTF8
+from __future__ import print_function, unicode_literals
+
+import os
+import re
+from collections import defaultdict
+
+from syncano.models import fields
+from syncano.models.incentives import Script
+
+from . import LOG, mute_log
+
+
+ALLOWED_RUNTIMES = {
```

Comment:
It's mostly used for getting file extension, I will remove the check. But there is also a problem with syncano-python that validates to much on client side (eg runtime_name validator is outdated).